src/vmspawn/vmspawn.c

```diff
@@ -775,6 +785,22 @@ static int run_virtual_machine(void) {
                         return log_error_errno(r, "Failed to create runtime directory: %m");
         }
 
+        switch (arg_qemu_net) {
+        case QEMU_NET_NONE:
+                r = strv_extend_strv(&cmdline, STRV_MAKE("-nic", "none"), /* filter_duplicates= */ false);
+                break;
+        case QEMU_NET_USER:
+                r = strv_extend_strv(&cmdline, STRV_MAKE("-nic", "user,model=virtio-net-pci"), /* filter_duplicates= */ false);
+                break;
+        case QEMU_NET_TAP:
```

Comment:
We should bail if we're not running as root and this is selected as qemu will just fail with a permission error

Reply:
We should also mention in the man page that this requires a correctly configured systemd-networkd to be running on the host so the host interface gets configured correctly.

Reply:
strv_extend_many()